concent_api/common/decorators.py

```diff
@@ -47,3 +52,15 @@ def wrapper(*args: Any, **kwargs: Any) -> None:
                 logging_level=LoggingLevel.ERROR)
             raise
     return wrapper
+
+
+def non_nesting_atomic(using: Union[str, Callable]=None, savepoint: bool=True) -> Atomic:
```

Comment:
the typing is still not ok here... the default value for `using` is neither `str`, nor `Callable`

Reply:
:+1: done